CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ All notable changes to this project will be documented in this file.
 -   #682 : Wrong data layout when copying a slice of an array.
 -   #1453 : Fix error-level developer mode output.
 -   \[INTERNALS\] Fix string base class selection.
+-   #1502 : Allowing the bot to re-run cancelled tests.
```

Comment:
Changes to the bot do not need to appear in the CHANGELOG. The CHANGELOG is about changes in the code. It serves as a reference for users to decide which version of Pyccel they need to use. Changes to the bot do not affect users and developers can't choose a version of the bot to use for their PRs so this information is not useful here

Reply:
Thank you, for your review; Fixed.